### PR TITLE
Added v1.3 release notes

### PR DESCRIPTION
**Pull request by bogdanromanx**

### CODE REVIEW

File: src/main/paradox/docs/releases/index.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 @@@ index
 
+* [v1.2 To v1.3 Migration](v1.2-to-v1.3-migration.md)
```
Comment (umbreak) [inline src/main/paradox/docs/releases/index.md]: Why is this the first item and not `v1.3 Release notes`?

At the bottom be have `v1.1 Release Notes` and after `v1.0 To v1.1 Migration`

File: src/main/paradox/docs/releases/index.md [hunk 2]
```diff
@@ -11,7 +13,26 @@
...
+* Introduced a @ref:[new type of view](../api/current/kg/views/composite-view-api.md) (_CompositeView_, currently as a
+  Beta feature) that expands on the indexing capabilities of the system through the ability to consume multiple sources
+  (multiple projects in the same Nexus deployment and projects in different Nexus deployments);
+* Added the ability to generate tabular views on the data available to a specific project (using any SparqlView defined
+  in the project - default SparqlView or AggregateSparqlViews) by means of Studios and Dashboards in Nexus Web;
+* Allow querying SparqlViews using the GET http method;
+* Exposed a new view subresource `.../offset` that presents the current view offset, or collection of offsets in case
+  of CompositeViews. The offset has the same value used with Server Sent Events as means of keeping track of the current
+  event replay progress. Deleting this resource with instruct the system to rebuild the indices of the selected view. 
+
```
Comment (umbreak) [inline src/main/paradox/docs/releases/index.md]: Worth adding here the [sort query parameter](https://github.com/BlueBrain/nexus/issues/924) on the listing API

Also worth mentioning the improve of the `q` query parameter on listings API

File: src/main/paradox/docs/releases/v1.3-release-notes.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+The system acknowledges the command by providing a `NoOffset` response:
+
+```json
+{
+    "@context": "https://bluebrain.github.io/nexus/contexts/offset.json",
+    "@type": "NoOffset"
+}
+```
+
```
Comment (umbreak) [inline src/main/paradox/docs/releases/v1.3-release-notes.md]: There should be a mention to the endpoints

- GET `/v1/views/{org_label}/{project_label}/{view_id}/projections/_/offset`
Fetch the projections offset

- DELETE `/v1/views/{org_label}/{project_label}/{view_id}/projections/_/offset`
rebuild the projections, without rebuilding the temporary blazegraph query space. This allows for rebuilding the projections with potentially greater amount of triples in the temporary blazegraph space

Also, required mention of additions & improvements on listings API:
- Added `sort` query param.
- Improved the results from `q` query param.

File: src/main/paradox/docs/releases/index.md (hunk 1 above)
Comment (bogdanromanx) [reply src/main/paradox/docs/releases/index.md]: Done.

File: src/main/paradox/docs/releases/index.md (hunk 2 above)
Comment (bogdanromanx) [reply src/main/paradox/docs/releases/index.md]: Done.

File: src/main/paradox/docs/releases/v1.3-release-notes.md (hunk 3 above)
Comment (bogdanromanx) [reply src/main/paradox/docs/releases/v1.3-release-notes.md]: Done.

**Pull request merged**